tests/unittests/OperatorTest.cpp

```diff
@@ -2632,6 +2632,57 @@ TEST_P(InterpAndCPU, QuantizedTile) {
   }
 }
 
+TEST_P(InterpAndCPU, ClipWithDefaultValue) {
```

Comment:
There's no need for this test, so you can delete it.

The default values are provided by the loader, not by the graph node itself, so this is not testing anything that the test below is not.

Reply:
yes, I agree, graph structure is the same.